syntax = "proto3";

message StartStream {

}
